src/routes/(app)/product/[id]/+layout.server.ts

```diff
@@ -0,0 +1,98 @@
+import { collections } from '$lib/server/database';
+import { error } from '@sveltejs/kit';
+import { cmsFromContent } from '$lib/server/cms';
+import type { Product } from '$lib/types/Product';
+import { omit } from 'lodash-es';
+
+export const load = async ({ params, locals }) => {
+	const product = await collections.products.findOne<
```

Comment:
instead of `.findOne` you can just `.count({_id: ...})` to see if the product exists

You certainly don't need the whole projection.